src/main/java/com/mraof/minestuck/client/model/UnderlingModel.java

```diff
@@ -3,11 +3,10 @@
 import com.mraof.minestuck.Minestuck;
 import com.mraof.minestuck.entity.underling.UnderlingEntity;
 import net.minecraft.util.ResourceLocation;
-import software.bernie.geckolib3.model.AnimatedGeoModel;
 
 import java.util.Objects;
 
-public class UnderlingModel<T extends UnderlingEntity> extends AnimatedGeoModel<T>
+public class UnderlingModel<T extends UnderlingEntity> extends AnimatedModel<T>
```

Comment:
While it would be most applicable elsewhere, I think it would be easy to forget if not preserved in code. The ogres attack animation involves its head bone, which causes it to awkwardly look directly forward or rotate when attacking

Reply:
This is an animation fix however, so having seen its implementation elsewhere it looks all good to move forward with

Reply:
Riotmode has been made aware of the issue and I will put a note in Geckolib that it still needs fixed to avoid requiring commenting here